.rustfmt.toml

```diff
@@ -0,0 +1,13 @@
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
```

Comment:
100 is the default.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#max_width